ansible/roles/ocp4-workload-mlops/tasks/setup-users.yaml

```diff
@@ -0,0 +1,63 @@
+---
+- name: Create ansible image image stream
+  k8s:
+    state: present
+    definition: "{{ lookup('file', 'ansible_pb_ist.yaml' ) | from_yaml }}"
+
+- name: Create ansible image build config
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'ansible_pb_bc.yaml.j2' ) | from_yaml }}"
+
+- name: Create ansible service account
+  k8s:
+    state: present
+    definition: "{{ lookup('file', 'ansible_sa.yaml' ) | from_yaml }}"
+
+- name: Assign cluster-admin custer role-based binding for ansible sa
+  k8s:
+    state: present
+    definition: "{{ lookup('file', 'ansible_admin_crb.yaml' ) | from_yaml }}"
+
+- name: Start ansible image build and wait for it to finish
+  command: oc start-build ansible-pb-build -n labs-setup -w
```

Comment:
Can we drop the `-w` and add a task to watch for the finish of the build...assuming you are trying to use the `-w` to force Ansible to watch and wait for the build to complete?